client/state/sites/test/reducers.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { RECEIVE_SITE } from '../action-types';
+import { items } from '../reducers';
+
+describe( 'reducers', () => {
+	describe( '#items()', () => {
+		it( 'should default to an empty object', () => {
+			const state = items( undefined, {} );
+
+			expect( state ).to.eql( {} );
+		} );
+
+		it( 'should index sites by ID', () => {
+			const state = items( null, {
+				type: RECEIVE_SITE,
+				site: { ID: 2916284, name: 'WordPress.com Example Blog' }
+			} );
+
+			expect( state ).to.eql( {
+				2916284: { ID: 2916284, name: 'WordPress.com Example Blog' }
+			} );
+		} );
+
+		it( 'should return a new instance of the state object', () => {
+			const original = {};
+			const state = items( original, {
+				type: RECEIVE_SITE,
+				site: { ID: 2916284, name: 'WordPress.com Example Blog' }
+			} );
+
+			expect( state ).to.not.equal( original );
+		} );
+
+		it( 'should accumulate sites', () => {
+			const state = items( {
```

Comment:
We should put the initial state on a separate variable as in the precedent test.
I think we could check `deep-freeze`( https://github.com/substack/deep-freeze ) on it as a good practice when testing reducers.